src/app/shared/ui/confirm-dialog/confirm-dialog.component.ts

```diff
@@ -8,8 +8,11 @@ import { Component, inject } from '@angular/core';
 import { MatButtonModule } from '@angular/material/button';
 import {
   MAT_DIALOG_DATA,
+  MatDialogActions,
```

Comment:
Are these dialogs really necessary? Are they not available by just importing the MatDialogModule?
It's a minor thing anyways.

Reply:
Do you mean the imports here or generally having a specialized confirmation dialog?

I think the imports are necessary because we use the  mat-dialog-actions directives etc. And I added the specialized confirmation dialog because we need this kind of dialog frequently, so having a shared component is handy and also ensures a consistent appearance.

Reply:
I meant the imports yes sorry.